client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import PurchaseDetail from './purchase-detail';
+
+const BusinessPlanDetails = ( { isFreeTrial, selectedSite } ) => {
+	var showGetFreeDomainTip = ! isFreeTrial;
```

Comment:
I'm not sure we need to review this code again, plus it's probably going to be modified in the next few days ;)